README.md

```diff
@@ -136,6 +136,84 @@ object Main extends App {
 }
 ```
 
+##### Document Streaming
+
+Fauna supports document streaming, where changes to a streamed document are pushed to all clients subscribing to that document.
+
+The following sections provide examples for managing streams with Flow or Monix, and
+assume that you have already created a `FaunaClient`.
+
+###### Flow subscriber
+
+It is possible to use the [java.util.concurrent.Flow](https://docs.oracle.com/javase/9/docs/api/java/util/concurrent/Flow.html) API directly by binding a `Subscriber` manually.
+
+In the example below, we are capturing the 4 first messages:
+
+```scala
+// docRef is a reference to the document for which we want to stream updates.
```

Comment:
Could we include the main imports necessary for running the example at the top of the code snippet? :)

```scala
import faunadb._
import faunadb.query._
```